tests/dags/providers/provider_api_scripts/test_brooklyn_museum.py

```diff
@@ -1,189 +1,164 @@
 import json
-import logging
-import os
-from unittest.mock import MagicMock, patch
+from pathlib import Path
+from unittest.mock import patch
 
-import requests
-from providers.provider_api_scripts import brooklyn_museum as bkm
+import pytest
+from common.licenses import LicenseInfo
+from common.loader import provider_details as prov
+from common.storage.image import ImageStore
+from providers.provider_api_scripts.brooklyn_museum import BrooklynMuseumDataIngester
 
 
-RESOURCES = os.path.join(
-    os.path.abspath(os.path.dirname(__file__)), "resources/brooklynmuseum"
-)
-
-logging.basicConfig(
-    format="%(asctime)s - %(name)s - %(levelname)s:  %(message)s", level=logging.DEBUG
-)
+RESOURCES = Path(__file__).parent / "resources/brooklynmuseum"
+bkm = BrooklynMuseumDataIngester()
+image_store = ImageStore(provider=prov.BROOKLYN_DEFAULT_PROVIDER)
+bkm.media_stores = {"image": image_store}
 
 
 def _get_resource_json(json_name):
-    with open(os.path.join(RESOURCES, json_name)) as f:
-        resource_json = json.load(f)
-    return resource_json
+    return json.loads((RESOURCES / json_name).read_text())
```

Comment:
This is really nice! It would be nice to extract because we are using (and re-writing) this functionality in ever provider script test.

Reply:
I've thought the same thing 😅 I'll make an issue for it!

Reply:
Made: https://github.com/WordPress/openverse-catalog/issues/712